data/json/items/comestibles/raw_veggy.json

```diff
@@ -82,6 +82,7 @@
     "type": "COMESTIBLE",
     "id": "jerusalem_artichoke",
     "name": { "str": "Jerusalem artichoke" },
+    "comestible_type": "FOOD",
```

Comment:
```suggestion
```

Reply:
Ugh, looks like the result of the rebase. Good catch.